cmd/aad-cli/cli/user_test.go

```diff
@@ -145,12 +155,19 @@ func TestUserSetAttribute(t *testing.T) {
 			}
 
 			cacheDir := t.TempDir()
-			cacheDB := "db_with_old_users"
+			cacheDB := "db_with_expired_users"
 			testutils.PrepareDBsForTests(t, cacheDir, cacheDB)
 			cache := testutils.NewCacheForTests(t, cacheDir)
 			c := cli.New(cli.WithCache(cache), cli.WithCurrentUser("futureuser@domain.com"))
 
-			_, err := testutils.RunApp(t, c, args...)
+			// Gets the user time before running the cli
+			var wantTime time.Time
+			aux, err := cache.GetUserByName(context.Background(), username)
+			if err == nil {
```

Comment:
It seems the only case the user does not exist is with "no user". So I suggest that you populate the time if `username != nouser@domain.com` and assert that there is no error while trying to retrieve the time. (this is to avoid false positive if the test pass without having a time where we didn’t expect it)